docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+    wildduck:
+        network_mode: host
+        build: .
+        ports:
+            - "8080:8080"
+        depends_on:
+            - redis
+            - mongo
+
+    redis:
```

Comment:
Redis and mongo need to be an external container. Is this meant to be an example?

Reply:
Yes, the docker compose was meant to be an example, to get a minimal system up